check_tests.sh

```
@@ -0,0 +1,21 @@
+#!/bin/bash
+missing=()
+
+for file in $(find . -type f -name "*.go" ! -path "./unit_tests/*" ! -name "*_test.go"); do
```

Shellcheck (0.11.0)

[warning] For loops over find output are fragile. Use find -exec or a while read loop.

(SC2044)